core/templates/dev/head/pages/exploration_editor/simple_editor_tab/ScrollSyncService.js

```diff
@@ -76,7 +75,12 @@ oppia.factory('ScrollSyncService', [
       },
       // Scroll to an element in the editor given its id.
       scrollTo: function(id) {
-        $anchorScroll(id);
+        var element = angular.element($document[0].getElementById(id));
+        if (element && element.offset()) {
+          $('body').animate({
```

Comment:
The animation looks good but the scroll is not working on firefox. The browsers Firefox & IE utilize the html portion of this selector, but looks out for possible effects when doing that.